test/cmdlineTests/model_checker_show_unproved_true_all_engines/err

```diff
@@ -4,9 +4,4 @@ Warning: CHC: Assertion violation might happen here.
 10 |         assert(s.x > 0);
    |         ^^^^^^^^^^^^^^^
 
-Warning: BMC: Assertion violation might happen here.
-  --> model_checker_show_unproved_true_all_engines/input.sol:10:9:
-   |
-10 |         assert(s.x > 0);
-   |         ^^^^^^^^^^^^^^^
-Note:
+Info: BMC: 1 verification condition(s) proved safe! Enable the model checker option "show proved safe" to see all of them.
```

Comment:
Do you know why this test, and some others, now does not fail in that assert?

Reply:
Note that previously, the assert was not failing, just the solver was not able to prove that it does not fail.
If you look at the actual input, the assertion there is indeed safe.
Sometimes the solver's behavior changes in unexpected ways even with a small change in the encoding. This is one of those cases.